jarviscli/plugins/voice_control.py

```diff
@@ -44,18 +69,23 @@ def hear(jarvis, s):
             continue  # For ignoring the unreconized words error
 
     while listen is True:
-        print("Say somthing")
+        print("Say something")
         try:
             with sr.Microphone() as source:
                 r.adjust_for_ambient_noise(source)
                 audio = r.listen(source)
-                pinger = r.recognize_google(audio).lower()
+
+                if connected:
+                    pinger = r.recognize_google(audio).lower()
+                
+                else:
+                    pinger = r.recognize_sphinx(audio).lower()
 
             if (pinger == "stop"):
                 listen = False
                 print("Listening stopped.")
                 _jarvis.speech.text_to_speech("Listening stopped.")
-                break
```

Comment:
Why remove this break?

Reply:
the while loop is only active as long as `listen = True`. In that code block you're setting listen to False therefore it will break the loop, so there is no point in that break